### ISSUE TITLE
which version is this Discrete SAC based upon

### ISSUE THREAD

**Issue by junhuang-ifast** [keiohta/tf2rl#55]
hi, may I know which paper(s)/method the discrete SAC is based upon? 

from my understanding there are 3 main implementations of discrete SAC:

- Gumbel Softmax 
- KL Divergence
- Petros Christodoulou's 

but they all include the auto entropy tuning/temperature term. I may have missed it but I dont see it in your version of the code. Thanks for your time!

**Comment by keiohta** [comment]
Hi @junhuang-ifast, thank you for your comment!
My code is based on Petros Christodoulou's ([paper](https://arxiv.org/abs/1910.07207v1), [code](https://github.com/p-christ/Deep-Reinforcement-Learning-Algorithms-with-PyTorch)), but actually I haven't implemented auto temperature tuning yet.

I am implementing the function at [sac_discrete branch](https://github.com/keiohta/tf2rl/tree/sac_discrete).
Although it's not active now, I'll certainly implement that in near future. Thanks!

**Comment by junhuang-ifast** [comment]
I see. Thanks.
